prover/prover.md

```diff
@@ -68,6 +71,10 @@ module STRATEGIES-EXPORTED-SYNTAX
                         "in:" AxiomName ","
                         "vars:" VariableNames ","
                         "with:" Patterns ")"
+                    | "apply" "(" AxiomOrClaimName
```

Comment:
Please add comments to illustrate the meaning of the arguments (or refer to where the documentation is).